tests/e2e-cypress/integration/3d/transitionTo3d.cy.js

```diff
@@ -45,4 +45,29 @@ describe('Testing transitioning between 2D and 3D', () => {
             })
         })
     })
+    context('3D load', () => {
+        it('base layer load', () => {
+            cy.goToMapView()
+            // fixtures/256.jpeg will be fetched
+            cy.intercept('**/ch.swisstopo.swisstlm3d-karte-farbe.3d/**').as('baseLayer')
+            cy.get('[data-cy="3d-button"]').click()
+            cy.get('#cesium canvas').should('be.visible')
+            cy.get('#ol-map canvas').should('not.exist')
```

Comment:
if possible, it would be better to use custom `data-cy="..."` CSS selector (Cypress best practices), at least to grab the OL and Cesium map element (grabbing the canvas cannot be done any other way)

Reply:
updated